### DISCUSSION TITLE
Simplified Chinese (zh-cn) Language Leader

### DISCUSSION THREAD

**Discussion by al-one** [OHF-Voice/intents#888] [Language Leaders]
I would like to apply to be a Simplified Chinese language leader, it is my native language, and I have submitted a big PR to support the language.

- #678 
- #926 
- #971 
- #1009

**Comment by goxofy** [comment]
UP

**Comment by neroxps** [comment]
up

**Comment by lhlxx2010** [comment]
UP!!!!!!!!!!!!

**Comment by suifeng009** [comment]
up！

**Comment by lawis** [comment]
al-one we call him aLong  is a great contributor in the Chinese community of HA 。He gave me a lot of help in the process of using HA to build smart home. I really appreciate him.

**Comment by Maozhenlei** [comment]
He is Chinese HA no.1
